routes/api/roles/roles.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import Roles from '../../../controllers/roles';
+import auth from '../../../middlewares/auth';
+import Role from '../../../middlewares/roles';
```

Comment:
I suggest renaming the `Role` to `RoleMiddleware` and `Roles` to `RoleController`, to make them clear